addons/core/translations/resources/en-us.yaml

```diff
@@ -474,6 +474,7 @@ credential-library:
     key_type:
       label: Key Type
       help: Specifies the desired key type to use when generating a private key.
+      initial_option: Choose a type
```

Comment:
This seems like it would have wide applicability.

Reply:
Moved to the more generic forms folder instead